### ISSUE TITLE
Worth Testing on Windows

### ISSUE THREAD

**Issue by gauntface** [samccone/The-cost-of-transpiling-es2015-in-2016#29]
I just set up tests for Web Starter Kit on Appveyor and was pretty shocked at the difference in build time.

Rollup + Babel on my linux desktop is 3s.
Rollup + Babel on my Windows Surface is 30s
Rollup + Babel on Appveyor is 13s

Would be interesting to see if the results of tooling change based on platform.

**Comment by samccone** [comment]
yeah tool time is we relative to the system always --- so doing a 1:1 comparison between times for windows / linux will not say much given the different system stats

If i reworked the stat to be relative to each other in a column per platform... that would be interesting... :)

**Comment by paulirish** [comment]
gaunt and I spent some time trying to get appveyor to run the build script, but windows is very unfriendly to a lot of this.  

we got the basics working, though it'll require a fairly custom build file for windows. matt has some stuff moving on this.

**Comment by gauntface** [comment]
For current status.

Irish got something working along the lines of:

```
Measure-Command{babel ../src/src --presets ./node_modules/babel-preset-es2015 --out-dir=dist; browserify dist/app.js | ./node_modules/.bin/uglifyjs --compress --mangle > ../src/dist/bundle.js}
```

I'm probably going to try and wrap it in a node script just to run each tool.

**Comment by samccone** [comment]
just curious, the goal here --- is it just to get windows compile time?

(and yeah the makefile approach will 100% fail on windows.. as will most of the path stuff that is hardcoded)

Another thing to remember is the time number is going to be relative to the system stats... 

---

I did not really use the time per tool as an argument in my analysis because I knew this was going to be impossible to reproduce on other systems. if we wanted to  actually use these numbers we would want to run the tools 100% on CI and dump the numbers from there

**Comment by muloka** [comment]
I wonder if there's any noticeable differences between Node 5 vs Node 4.

**Comment by gauntface** [comment]
The actual time isn't important, as you said it's relative.

The main curiosity is whether some tools are faster on windows compared OS X or Linux. Lost cause?

**Comment by paulirish** [comment]
once we have the build pipeline working on windows, we'll be able to test on the same hardware. 
but getting some numbers is the first step towards refining the conclusion.
